server/src/test/scala/org/http4s/server/staticcontent/FileServiceSuite.scala

```diff
@@ -161,7 +161,7 @@ class FileServiceSuite extends Http4sSuite with StaticContentShared {
   }
 
   test("return files included via symlink") {
-    val relativePath = "symlink/org/http4s/server/staticcontent/FileServiceSuite.scala"
+    val relativePath = "symlink/org/http4s/server/staticcontent/StaticContentShared.scala"
```

Comment:
I quite misapprehend why, but that test ("return files included via symlink") is failing with added type annotations. This particular change actually fixes that test.

Reply:
Yeah I was wondering about this too. It would be good to understand why...

Reply:
Unfortunately, I even don't have any ideas about it. I was thinking about a kinda caching problem. But the test file is consistent when SBT is starting and when the test class is running. So if someone has any ideas, it'd be nice to figure out this a bit odd behavior.

Reply:
I'll try and take a look later, but no need to block this PR. 👍 from me, would be great to get @bplommer's eyes on it :)